src/actions/arcActions.ts

```diff
@@ -664,7 +666,6 @@ export function stakeProposal(daoAvatarAddress: string, proposalId: string, pred
       if(amount < minimumStakingFee) throw new Error(`Staked less than the minimum: ${minimumStakingFee}!`);
       if(amount > balance) throw new Error(`Staked more than than the balance: ${balance}!`);
 
-      await stakingToken.approve(votingMachineInstance.address, amount);
```

Comment:
why did you remove this? wasn't this necessary?

Reply:
Arc.js takes care of this now.